src/pages/sitemap-docs.xml.tsx

```diff
@@ -0,0 +1,31 @@
+import { GetServerSidePropsContext } from 'next'
+
+export default function SiteMap() {
+	return null
+}
+
+const siteUrl = process.env.SITE_URL || 'https://developer.hashicorp.com'
```

Comment:
We can use our config value, defined [here](https://github.com/hashicorp/dev-portal/blob/main/config/base.json#L22)

```suggestion
const siteUrl = __config.dev_dot.canonical_base_url
```

Reply:
Added thanks!